### ISSUE TITLE
Warning RecyclerView adapter Null Reference.

### ISSUE THREAD

**Issue by monday8am** [toggl/mobile#748]
XI: 15f93297f46b1b49051973dd3aa7090206c47ab2
System.NullReferenceExceptionObject reference not set to an instance of an object

```
  at Toggl.Joey.UI.Adapters.LogTimeEntriesAdapter.CollectionChanged (System.Collections.Specialized.NotifyCollectionChangedEventArgs e) [0x00000] in <filename unknown>:0 
  at Toggl.Joey.UI.Adapters.RecycledDataViewAdapter`1+UpdateScheduler[System.Object].RunUpdate (System.Collections.Specialized.NotifyCollectionChangedEventArgs eventInfo) [0x00000] in <filename unknown>:0 
```

**Comment by monday8am** [comment]
Waiting for evolution on XI.

**Comment by monday8am** [comment]
fixed with version 8.0.1
